security-policies/bundle/compliance/cis_azure/rules/cis_1_23/rule.rego

```diff
@@ -0,0 +1,22 @@
+package compliance.cis_azure.rules.cis_1_23
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+import future.keywords.every
+
+finding = result {
+	# filter
+	data_adapter.is_role_definition
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(no_custom_roles),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+no_custom_roles {
+	every role_def in data_adapter.role_definitions {
+		role_def.properties.type != "CustomRole"
+	}
+} else = false
```

Comment:
Need to understand this deeper but are you sure that this is equivalent to the CIS instructions?

## Instructions

### From Azure Portal
1. From Azure Home select the Portal Menu.
2. Select Subscriptions.
3. Select Access control (IAM).
4. Select Roles.
5. Click Type and select CustomRole from the drop down menu.
6. Select View next to a role.
7. Select JSON.
8. Check for assignableScopes set to / or the subscription, and actions set to *.
9. Repeat steps 6-8 for each custom role.

### From Azure CLI

List custom roles:
```
az role definition list --custom-role-only True
```
Check for entries with assignableScope of / or the subscription, and an action of *